README.md

```diff
@@ -1,70 +1,46 @@
-# Getting Started with Create React App
+## class 01 1/10/22
+- added basic components: Main, Footer, Header, and HornedBeasts Component(this renders the individual pictures with their descriptions)
+- HornedBeasts component is built out with an img tag with src, alt, and title attributes and a p tag
+  - the component takes in data for the tags and attributes via props passed from the main component
+- Main then invokes the Horned Beasts three times with the props hard coded in
```

Comment:
Oh! We just need the collaborators section in here and then we are good 👍  If you merge before seeing this, add it in on next lab.